openvpn-install.sh

```diff
@@ -596,8 +599,12 @@ function installOpenVPN () {
 		PASS=${PASS:-1}
 		CONTINUE=${CONTINUE:-y}
 
-		# Behind NAT, we'll default to the publicly reachable IPv4.
-		PUBLIC_IPV4=$(curl ifconfig.co)
+		# Behind NAT, we'll default to the publicly reachable IPv4/IPv6.
+		if [[ $IPV6_SUPPORT == "y" ]]; then
+			PUBLIC_IPV4=$(curl ifconfig.co)
+		else
+			PUBLIC_IPV4=$(curl -4 ifconfig.co)
+		fi
 		ENDPOINT=${ENDPOINT:-$PUBLIC_IPV4}
```

Comment:
@angristan are we sure we can change this behavior to default to an IPv6 endpoint?

Reply:
I think most people would still prefer to connect to an IPv4 endpoint...

Reply:
I prefer also an IPv4 endpoint.
An IPv6 endpoint should only a fallback, if IPv4 is not usable.

For my understand the IPV6_SUPPORT is a define to support IPv6 and NAT inside the VPN.
For a dual stack endpoint I enable IPV6_SUPPORT and put a FDNS name as endpoint.

Reply:
curl ifconfig.co in the current version returns an IPv6 address on my servers, which is why I made this change - although my servers are IPv6 enabled, I'm disabling IPv6 in OpenVPN (and forcing an IPv6 blackhole with route-ipv6 2000::/3 dead::beef). I could set PUBLIC_IPV4 separately from the script, so this isn't a make-or-break change for me, but it was an easy fix.

Reply:
@jmeubank Can you please change it to `https://ifconfig.co` to use the https version? You are already working on this lines so I'm not going to open a new PR.

Reply:
No prob - done.